weave/tests/test_client_trace.py

```diff
@@ -702,6 +703,374 @@ def basic_op(in_val: dict, delay) -> dict:
         assert inner_res.calls[2].inputs["in_val"]["prim"] == last
 
 
+def test_trace_call_filter(client):
+    is_sql_lite = isinstance(client.server, SqliteTraceServer)
+
+    @weave.op()
+    def basic_op(in_val: dict, delay) -> dict:
+        import time
+
+        time.sleep(delay)
```

Comment:
Could you add a comment for why there is a sleep.

Reply:
fixed on next commit - removed - this was leftover from sort's version (used to ensure log time is slightly different)